org.techxartisan.openterfaceQT/org.techxartisan.openterfaceQT.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: org.techxartisan.openterfaceQT
+runtime: org.kde.Platform
+runtime-version: "6.7"
+sdk: org.kde.Sdk
+command: openterfaceQT
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=all
+  - --filesystem=/run/udev:ro
+
+modules:
+  - name: libusb
```

Comment:
use it from shared modules https://github.com/flathub/shared-modules?tab=readme-ov-file#adding